examples/tf_cnn_benchmarks/tf_cnn_config.yaml

```diff
@@ -7,7 +7,7 @@ jobParams:
   args: "python,tf_cnn_benchmarks.py,--local_parameter_device=gpu,--device=cpu,--batch_size=4,--model=resnet50,--data_format=NHWC,--forward_only"
   numMasters: 0
   numPs: 1
-  numWo rkers: 1
+  numWo rkers: 3
```

Comment:
does this patch actually work for multiple workers?

Reply:
For now it's working only for one, you are right.